openwisp_controller/connection/base/models.py

```diff
@@ -44,6 +44,12 @@ def clean(self):
         self._validate_connector_schema()
 
     def _get_connector(self):
+        connectors = dict(app_settings.CONNECTORS)
```

Comment:
Maybe we can make this a class attribute so it is not evaluated every time this method is called.